.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ references:
 
   node_container: &node_container
     docker:
-      - image: circleci/node:6.11
+      - image: circleci/node:8.9
```

Comment:
i have some concerns about testing only on node 8, while we still technically support node 4. that being said, i'd like to keep pushing all customers to use node 8. so, that might a risk worth taking.